src/components/CategoryInput/AwardList.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import './AwardList.css';
+import ListItem from './ListItem';
+
+class AwardList extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+        currentAward: "",
+        awardsList: []
+    };
+    this.inputChange = this.inputChange.bind(this);
+    this.addToList = this.addToList.bind(this);
+  }
+
+  inputChange(event) {
+    this.setState({
+        currentAward: event.target.value
+    })
+  }
+
+  addToList(event) {
+    if (event.key == 'Enter') {
+        if (this.state.curr_item !== "") {
+            this.setState({
+                currentAward: "", 
+                awardsList: this.state.awardsList.concat(this.state.currentAward)
+            })
+            console.log("successfully added");
+        }
+    }
+  }
+
+  render() {
+    return (
+        <div className="AwardList">
+            <div className="list-name">
+                <h1>{this.props.listTitle}</h1>
+                <div className="blue-highlight"></div>
```

Comment:
Is this only for highlighting the `h1` blue. Why not just put the css background on the `h1`? If you need this div to have a blue background thats offset in some way I would recommend looking up what is called a css psuedo-element like `::before` or `::after`. This will allow you to have less markup that is solely for styling. One thing that is important to remember is that jsx/html is your "content" and the structure of your "content". Try to avoid having extra html/jsx for solely styling.